Pipfile

```diff
@@ -11,7 +11,7 @@ plotly = "*"
 sqlalchemy = "<2"
 sqlmodel = "*"
 statsmodels = "*"
-streamlit = "1.17.0"
+streamlit = "==1.17"
```

Comment:
not sure why this should be needed, but for me, the previous variant didn't work locally and it installed the latest streamlit version instead.